packages/editor-ui/src/utils/executionUtils.test.ts

```diff
@@ -1,23 +1,13 @@
 import { describe, it, expect, vi, beforeEach } from 'vitest';
-import {
-	displayForm,
-	openFormPopupWindow,
-	executionFilterToQueryFilter,
-	waitingNodeTooltip,
-} from './executionUtils';
+import { displayForm, executionFilterToQueryFilter, waitingNodeTooltip } from './executionUtils';
 import type { INode, IRunData, IPinData } from 'n8n-workflow';
 import { type INodeUi } from '../Interface';
+import { CHAT_TRIGGER_NODE_TYPE, FORM_TRIGGER_NODE_TYPE } from '@/constants';
+import { createTestNode } from '@/__tests__/mocks';
 
-const FORM_TRIGGER_NODE_TYPE = 'formTrigger';
 const WAIT_NODE_TYPE = 'waitNode';
 
-vi.mock('./executionUtils', async () => {
-	const actual = await vi.importActual('./executionUtils');
-	return {
-		...actual,
-		openFormPopupWindow: vi.fn(),
-	};
-});
+const windowOpenSpy = vi.spyOn(window, 'open');
```

Comment:
Replacing the spy target, because it seemed that `displayForm` function always calls the actual implementation of `openFormPopupWindow`